### ISSUE TITLE
Call to a member function resize() on a non-object

### ISSUE THREAD

**Issue by kaladivo** [fredrikekelund/grav-plugin-resize-images#9]
When I try to save page with uploaded image I get following error:

```
Whoops\Exception\ErrorException thrown with message "Call to a member function resize() on a non-object"

Stacktrace:
#0 Whoops\Exception\ErrorException in /Users/kaladivo/Downloads/grav-admin/user/plugins/resize-images/resize-images.php:72
```

**Comment by kaladivo** [comment]
It works fine when I change Image editing module. I think the error should be handled better.

**Comment by JohnMica** [comment]
it seems there is a similar error while locally / live site saving any pages as per the guys @ grav https://github.com/getgrav/grav/issues/1305 seems they know better then me what and how

**Comment by fredrikekelund** [comment]
Thanks for reporting this @kaladivo! This issue has been around for a little while and seems to have been due to some bad logic around picking the right image resizing adapter. It should be fixed in the 0.2.0 release (which I just pushed to GitHub). Let me know if you keep seeing this issue with the latest version as well!
